aiida/restapi/translator/nodes/data/upf.py

```diff
@@ -41,36 +40,12 @@ def __init__(self, **kwargs):
         super(UpfDataTranslator, self).__init__(Class=self.__class__, **kwargs)
 
     @staticmethod
-    def get_visualization_data(node, visformat=None):
+    def get_derived_properties(node):
```

Comment:
Probably good idea to just have the base class raise `RestFeatureNotAvailable` so not every sub class has to do it if they do not provide it